docs/tctl.md

```diff
@@ -340,3 +322,99 @@ As you add the bad binary checksum to your namespace, Temporal will not dispatch
 `--tls_server_name=<server name>` command line argument that passes an override value for the target server that is used for TLS host verification. It also enables host verification. The value must be one of the DNS names listed in the server TLS certificate.
 
 TLS command line arguments can be provided via their respective environment variables to shorten the command line.
+
+## Search Workflows
+
+### List closed or open workflow executions
+
+```
+./tctl workflow list
+
+# default will only show one page, to view more items, use --more flag
+./tctl workflow list -m
+```
+
+Use **--query** to list workflows with SQL like query:
```

Comment:
I would love all of these to not require scroll bar.

Reply:
I did my best with this.
Each flag can be moved to a new line. But I am sure the command would break if the query itself is broken up, which is the longest part of the cmd. Might be able to do something creative - but then it would likely result in a decrease in user comprehension.